### PR TITLE
Implement department adapter

### PR DESCRIPTION
**Pull request by b1acKr0se**

### CODE REVIEW

**Comment by phamxuanlu** [comment]
OK

File: app/src/main/res/layout/item_department.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    android:foreground="?attr/selectableItemBackground">
+    <TextView
+        android:id="@+id/text_department_name"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center_vertical"
+        android:gravity="center_vertical"
+        android:paddingStart="@dimen/default_padding_8dp"
+        android:paddingEnd="@dimen/default_padding_8dp"
+        android:textSize="20sp"/>
```
Comment (trungkienvu85) [inline app/src/main/res/layout/item_department.xml]: dimen!

File: app/src/main/java/com/framgia/nguyenthanhhai/humanresourcemanagementsystem/ui/adapter/DepartmentAdapter.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            super(view);
+            mDepartmentNameTv = (TextView) view.findViewById(R.id.text_department_name);
+        }
+
+        @Override
+        public void onClick(View v) {
+            mDepartmentClickListener.onDepartmentClick(department);
+        }
+    }
+
```
Comment (trungkienvu85) [inline app/src/main/java/com/framgia/nguyenthanhhai/humanresourcemanagementsystem/ui/adapter/DepartmentAdapter.java]: blank line!

**Comment by trungkienvu85** [comment]
OK!

**Pull request merged**